### PR TITLE
Translation in Japanese

### PR DESCRIPTION
**Pull request by zukizukizuki**
Japanese translation was performed.

## Things did
- Copy `en` folder
- Translated values in Json from English to Japanese

## Notes
English that can be understood in Japan is left as it is.

## Issue
https://github.com/privacy-scaling-explorations/website-v2/issues/122

### CODE REVIEW

**Comment by zukizukizuki** [comment]
@ChialiT 
please check it.

**Comment by zukizukizuki** [comment]
@ChialiT 
How's it going?

**Comment by ChialiT** [comment]
> @ChialiT How's it going?

Thank you for pinging me, I asked one of our member to review the translation. my Japanese is very limited.

**Comment by tkmct** [comment]
Hi @zukizukizuki, thank you for your contribution. I'm currently reviewing your translation. Bear with me for a moment please.

File: app/i18n/locales/jp/programs-page.json [hunk 1]
```diff
@@ -0,0 +1,91 @@
+{
+  "title": "プログラム可能な暗号化を始めましょう (Get started with programmable cryptography)",
+  "description": "PSEは、ゼロ知識証明、完全準同型暗号、マルチパーティ計算、そして最新のプログラム可能な暗号について深く学ぶための無料の教育プログラムを提供しています。基礎を学びたい方でも、オープンソースへの貢献を始めたい方でも、学習の旅に役立つリソースを用意しています！ (PSE offers free educational programs to help you dive deep into Zero Knowledge Proofs, Fully Homomorphic Encryption, Multi-Party Computation and the latest in programmable cryptography. Whether you’re looking to learn the basics or start making open-source contributions, we have resources for your learning journey!)",
+  "common": {
+    "faq": "よくある質問 (FAQ)",
```
Comment (tkmct) [inline app/i18n/locales/jp/programs-page.json:5]:
```suggestion
    "faq": "よくある質問",
```

File: app/i18n/locales/jp/programs-page.json [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    "curriculum": "カリキュラム (Curriculum)",
```
Comment (tkmct) [inline app/i18n/locales/jp/programs-page.json:6]:
```suggestion
    "curriculum": "カリキュラム",
```

File: app/i18n/locales/jp/programs-page.json [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    "howToApply": "応募方法 (How to apply)",
```
Comment (tkmct) [inline app/i18n/locales/jp/programs-page.json:7]:
```suggestion
    "howToApply": "応募方法",
```

File: app/i18n/locales/jp/programs-page.json [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    "applyNow": "今すぐ応募する (Apply now)",
```
Comment (tkmct) [inline app/i18n/locales/jp/programs-page.json:8]:
```suggestion
    "applyNow": "今すぐ応募する",
```

File: app/i18n/locales/jp/programs-page.json [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+    "learnMoreOnGithub": "GitHubで詳細を見る (Learn more on Github)",
```
Comment (tkmct) [inline app/i18n/locales/jp/programs-page.json:9]:
```suggestion
    "learnMoreOnGithub": "GitHubで詳細を見る",
```

File: app/i18n/locales/jp/programs-page.json [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+    "week": "Week {{week}} (週 {{week}})"
```
Comment (tkmct) [inline app/i18n/locales/jp/programs-page.json:10]:
```suggestion
    "week": "第{{week}}週"
```

File: app/i18n/locales/jp/programs-page.json [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+  },
+  "curriculum": [
+    {
+      "title": "前提条件 (PRE-REQUISITES)",
```
Comment (tkmct) [inline app/i18n/locales/jp/programs-page.json:14]:
```suggestion
      "title": "必須条件",
```

File: app/i18n/locales/jp/programs-page.json [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+      "items": [
+        "コースの概要とリソース (Course overview and resources)",
```
Comment (tkmct) [inline app/i18n/locales/jp/programs-page.json:16]:
```suggestion
        "コースの概要とリソース",
```

File: app/i18n/locales/jp/programs-page.json [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+        "Git、GitHub、およびPRワークフローの基本 (Git, GitHub, and PR workflow basics)",
```
Comment (tkmct) [inline app/i18n/locales/jp/programs-page.json:17]:
```suggestion
        "Git、GitHub、およびPRワークフローの基本",
```

File: app/i18n/locales/jp/programs-page.json [hunk 10]
```diff
@@ -0,0 +1,91 @@
...
+        "ゼロ知識証明 (ZKPs) と数論の入門 (Introduction to ZKPs and Number Theory)"
```
Comment (tkmct) [inline app/i18n/locales/jp/programs-page.json:18]:
```suggestion
        "ゼロ知識証明 (ZKPs) と数論の入門"
```

File: app/i18n/locales/jp/programs-page.json [hunk 11]
```diff
@@ -0,0 +1,91 @@
...
+      ]
+    },
+    {
+      "title": "暗号の基本 (Cryptographic Basics)",
```
Comment (tkmct) [inline app/i18n/locales/jp/programs-page.json:22]:
```suggestion
      "title": "暗号学の基礎",
```

File: app/i18n/locales/jp/programs-page.json [hunk 12]
```diff
@@ -0,0 +1,91 @@
...
+      "items": [
+        "Circom入門 (Getting started with Circom)",
```
Comment (tkmct) [inline app/i18n/locales/jp/programs-page.json:24]:
```suggestion
        "Circom入門",
```

File: app/i18n/locales/jp/programs-page.json [hunk 13]
```diff
@@ -0,0 +1,91 @@
...
+        "暗号化とハッシュ関数の基礎 (Basics of encryption and hash functions)",
```
Comment (tkmct) [inline app/i18n/locales/jp/programs-page.json:25]:
```suggestion
        "暗号化とハッシュ関数の基礎",
```

File: app/i18n/locales/jp/programs-page.json [hunk 14]
```diff
@@ -0,0 +1,91 @@
...
+        "デジタル署名と楕円曲線暗号 (Digital signatures and elliptic curve cryptography)"
```
Comment (tkmct) [inline app/i18n/locales/jp/programs-page.json:26]:
```suggestion
        "デジタル署名と楕円曲線暗号"
```

File: app/i18n/locales/jp/programs-page.json [hunk 15]
```diff
@@ -0,0 +1,91 @@
...
+      ]
+    },
+    {
+      "title": "より高度な暗号化 + ZKP (More Crypto + ZKPs)",
```
Comment (tkmct) [inline app/i18n/locales/jp/programs-page.json:30]:
```suggestion
      "title": "より高度な暗号技術とZKP",
```

File: app/i18n/locales/jp/programs-page.json [hunk 16]
```diff
@@ -0,0 +1,91 @@
...
+      "items": [
+        "Circom 集中講座 + 実践 (Circom crash course + practice)",
```
Comment (tkmct) [inline app/i18n/locales/jp/programs-page.json:32]:
```suggestion
        "Circom 集中講座 + 実践",
```

File: app/i18n/locales/jp/programs-page.json [hunk 17]
```diff
@@ -0,0 +1,91 @@
...
+        "KZGコミットメントと zkSNARK (KZG Commitments and zkSNARKs)",
```
Comment (tkmct) [inline app/i18n/locales/jp/programs-page.json:33]:
```suggestion
        "KZGコミットメントと zkSNARKs",
```

File: app/i18n/locales/jp/programs-page.json [hunk 18]
```diff
@@ -0,0 +1,91 @@
...
+        "信頼されたセットアップと Groth16 の概要 (Overview of Trusted Setups and Groth16)"
```
Comment (tkmct) [inline app/i18n/locales/jp/programs-page.json:34]:
```suggestion
        "トラステッドセットアップと Groth16 の概要"
```

File: app/i18n/locales/jp/common.json [hunk 19]
```diff
@@ -0,0 +1,83 @@
+{
+  "siteTitle": "プライバシーとスケーリングの探究",
+  "siteDescription": "暗号研究と共同実験を通じてイーサリアムを強化する",
```
Comment (tkmct) [inline app/i18n/locales/jp/common.json:3]:
```suggestion
  "siteDescription": "暗号学研究とコレクティブな実験を通じてイーサリアムをよりよくする",
```

**Review by tkmct — CHANGES_REQUESTED** [review]
Thank you very much for your contribution. I added suggestions in the PR.
Please take a look and accept those suggestions. If you need any help, let us know.

**Comment by zukizukizuki** [comment]
@tkmct 
thank you for your review!
All imported.

File: app/i18n/locales/jp/about-page.json [hunk 20]
```diff
@@ -0,0 +1,39 @@
+{
+  "title": "チームについて",
+  "description": "PSEはイーサリアム財団が支援する研究開発ラボで、暗号プリミティブのユースケースを試すことに専念しています。私たちは、オープンソースのインフラやツールの開発、ワークショップやサマープログラムの開催、そして暗号学を現実のものにしようとする人々とのコラボレーションなどを行います。",
```
Comment (tkmct) [inline app/i18n/locales/jp/about-page.json:3]:
```suggestion
  "description": "PSEはイーサリアムファウンデーションが支援する研究開発ラボで、暗号プリミティブのユースケースを試すことに専念しています。私たちは、オープンソースのインフラやツールの開発、ワークショップやサマープログラムの開催、そして暗号学を現実のものにしようとする人々とのコラボレーションなどを行います。",
```

**Review by tkmct — CHANGES_REQUESTED** [review]
I'm sorry, I missed one change. Can you apply the suggested change too?

**Review by tkmct — APPROVED** [review]
Thanks! LGTM.

**Comment by zukizukizuki** [comment]
@tkmct (@ChialiT) 
thank you!Waiting for approval.

**Comment by AtHeartEngineer** [comment]
@tkmct Do you think this is ready to merge?

**Review by ChialiT — APPROVED** [review]

**Comment by ChialiT** [comment]
@AtHeartEngineer can u approve this one so i can merge it?thanks

**Review by AtHeartEngineer — APPROVED** [review]

**Pull request merged**